lib/resources/src/main-cli.js

```diff
@@ -1,3 +1,4 @@
+import '@babel/polyfill';
```

Comment:
I'm confused. Why is the CLI dependent on Babel?

Reply:
It's was turned on previously to enable #959. Before this fix, it was in prepend list for built-in bundler based app.

Reply:
The other fact is that we have babel polyfill in our webpack app skeleton from day1 I think.

Reply:
Apologies. I misread this code. This is a file that's being scaffolded into a new project, not the CLI source code itself. I was trying to understand why our CLI code directly needed babel, but it doesn't.

Reply:
Actually cli code does... :-)
The bundler use babel to support esm format.

Reply:
Ugh. Is there any way not to have a dependency on Babel?

Reply:
Actually it always have, even before auto-tracing.

cli uses babel to support tasks in exnext, uses typescript to support tasks in typescript.

I argued we should rewrite all tasks in pure commonjs, in order to remove babel and typescript from dependencies.

Reply:
But that (removing babel/ts) of-course is a breaking change.

Reply:
I know it uses it when you generate an ESNext project. Is there any way to not use it directly in the CLI source itself?

Reply:
Use babel to do just esm transform is an overkill, there definitely is some other smaller lib for just that. I can do some investigation.

For the tasks part, the babel and ts are runtime dependencies. And we previously don't have them directly in aurelia-cli package.json, but that created plenty of trouble for aurelia-cli developers. Remember those special part in README about npm install after npm link?

@JeroenVinke added those runtime deps (babel and ts) to aurelia-cli package.json, to ease the experience on developing aurelia-cli.

Reply:
Ah yes. I remember that issue now. Let's keep things as is then. Not a high priority to change this for now. I had just forgotten we had that dependency in the CLI itself.